cookbooks/view_pings_cep.md

```diff
@@ -0,0 +1,57 @@
+# See My Pings
+
+So you want to see what you're sending the telemetry pipeline, huh? Well follow these steps and we'll have you reading some JSON in no time.
+
+Note that some of these steps were stolen from the [Creating a Real-Time Analysis Plugin Cookbook](realtime_analysis_plugin.md).
```

Comment:
Maybe frame this as "For a more thorough introduction to real-time analysis, see <other thing>".